packages/backend/src/services/AsyncQueryService/AsyncQueryService.ts

```diff
@@ -694,13 +725,57 @@ export class AsyncQueryService extends ProjectService {
                 streams: { readStream: Readable; writeStream: Writable },
             ) => Promise<{ truncated: boolean }>;
         },
+        options?: {
+            onlyRaw?: boolean;
+            showTableNames?: boolean;
+            customLabels?: Record<string, string>;
+            columnOrder?: string[];
+            hiddenFields?: string[];
+            chartName?: string;
+            csvLimit?: number | null;
+            pivotConfig?: PivotConfig;
```

Comment:
We should remove `chartName` and `csvLimit` from the endpoint body as they are not used with the new flow. 

`pivotConfig` needs to be supported for now because we are not handling it in SQL for explorer charts. I don't see this property being used so I suspect there is a bug when the viz is a table with pivot.

Reply:
Confirmed it's not applying pivot

In `main` 
<img width="1884" alt="Screenshot 2025-06-03 at 16 16 39" src="https://github.com/user-attachments/assets/89706085-5c3c-446e-983b-0cb76959e38c" />

In branch
<img width="1885" alt="Screenshot 2025-06-03 at 16 17 07" src="https://github.com/user-attachments/assets/f488ee8e-34ae-4218-b016-6936dc2a082a" />